pmoves/services/wger-nats-bridge/main.py

```
@@ -0,0 +1,203 @@
+
+        self._save_watermark()
+
+        if new_workouts or new_weights:
+            logger.info("Poll: workouts=%d weights=%d measurements=%d", len(new_workouts), len(new_weights), len(new_meas))
+
+    async def run(self):
+        self.nc = NATS()
+        try:
+            await self.nc.connect(self.nats_url, connect_timeout=10)
+            logger.info("Connected to NATS at %s", _redact_url(self.nats_url))
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)